.gitignore

```diff
@@ -1,15 +1,52 @@
 *.pyc
-!.travis.yml
 *.swp
 *.lock
 *.log
 *.pid
+cms/django
 *.DS_Store
+*.svn
```

Comment:
Guys, I think there's been too much emphasis on cleaning up the .gitignore files. Remember that some of the items in here may be for certain build tools that only a single person uses. It should be OK for that person to add those things, if it helps them be productive.